tests/sentry/mediators/sentry_app_installations/test_destroyer.py

```diff
@@ -40,6 +42,16 @@ def test_deletes_grant(self):
 
         assert not ApiGrant.objects.filter(pk=grant.id).exists()
 
+    def test_deletes_service_hooks(self):
+        hook = ServiceHook.objects.get(
+            application=self.sentry_app.application,
+            actor_id=self.install.id,
+        )
```

Comment:
Just fyi, there's a `create_service_hook()` function.

Reply:
looking at it, shouldn't the actor be the install (in the case of sentry apps)? right now the `create_service_hook` creates a user to be the actor